src/eth1spec/rlp.py

```diff
@@ -351,3 +364,118 @@ def decode_item_length(encoded_data: Bytes) -> int:
         )
 
     return 1 + length_length + decoded_data_length
+
+
+#
+# Encoding and decoding custom dataclasses like Account, Transaction,
+# Receipt etc.
+#
+
+
+def encode_block(raw_block_data: Block) -> Bytes:
+    """
+    Encode `Block` dataclass
+    """
+    return encode(
+        (
+            raw_block_data.header,
+            raw_block_data.transactions,
+            raw_block_data.ommers,
+        )
+    )
+
+
+def encode_header(raw_header_data: Header) -> Bytes:
+    """
+    Encode `Header` dataclass
+    """
+    return encode(
+        (
+            raw_header_data.parent,
+            raw_header_data.ommers,
+            raw_header_data.coinbase,
+            raw_header_data.state_root,
+            raw_header_data.transactions_root,
+            raw_header_data.receipt_root,
+            raw_header_data.bloom,
+            raw_header_data.difficulty,
+            raw_header_data.number,
+            raw_header_data.gas_limit,
+            raw_header_data.gas_used,
+            raw_header_data.time,
+            raw_header_data.extra,
+            raw_header_data.mix_digest,
+            raw_header_data.nonce,
+        )
+    )
+
+
+def encode_account(raw_account_data: Account) -> Bytes:
```

Comment:
I'm a little worried to call this `encode_account` because it masks the patricialization happening. I think it is okay for now, but we'll definitely want to change in the future.